ci/release/update-version.sh

```diff
@@ -61,7 +61,7 @@ DEPENDENCIES=(
 )
 for FILE in dependencies.yaml conda/environments/*.yaml; do
   for DEP in "${DEPENDENCIES[@]}"; do
-    sed_runner "/-.* ${DEP}\(-cu[[:digit:]]\{2\}\)\{0,1\}==/ s/==.*/==${NEXT_SHORT_TAG_PEP440}.*/g" "${FILE}"
+    sed_runner "/-.* ${DEP}\(-cu[[:digit:]]\{2\}\)\{0,1\}==/ s/==.*/==${NEXT_SHORT_TAG_PEP440}.*,>=0.0.0a0/g" "${FILE}"
```

Comment:
oh wow nice! I would have missed this `update-version.sh` part of the update.

I tested like this on `cudf`:

```shell
bash ci/release/update-version.sh '24.10.00'
```

And you're definitely right... without this change, `update-version.sh` would drop the alpha spec.

I'll apply this change on the `cudf` PR (https://github.com/rapidsai/cudf/pull/1524) and check if it needs to be added for `rmm`, and do that on all the rest of the PRs I open.